.ebextensions/04_create_apache_conf.config

```diff
@@ -0,0 +1,30 @@
+#
+# Create an Apache .conf file for the preview service.
+#
+files:
+  '/tmp/suitec-preview-service.httpd.conf' :
+    mode: '000755'
```

Comment:
@paulkerschen another minor question: why not mode `000644`?  Seems more standard